docs/source/install.rst

```diff
@@ -54,56 +54,18 @@ The instructions above install a CPU-only version of SCICO. To install a version
 
    ::
 
-      pip install --upgrade "jaxlib==0.1.70+cuda110" -f https://storage.googleapis.com/jax-releases/jax_releases.html
+      pip install --upgrade "jaxlib==0.1.75+cuda110" -f https://storage.googleapis.com/jax-releases/jax_releases.html
 
 
 
 Additional Dependencies
 -----------------------
 
-We use the `ASTRA Toolbox <https://www.astra-toolbox.com/>`_ for tomographic projectors. We currently require the development version of ASTRA, as suggested by the package maintainers.
```

Comment:
The first sentence here seems to have been lost (rather than moved) in the edits.

Reply:
I removed it since it seemed that we are explaining why we need astra but not why we need the other example_requirements. Shall I add a few sentences to explain how we are using each of the example_requirements or just add back this sentence on astra?

Reply:
Fair point. I'm OK with leaving it as-is.